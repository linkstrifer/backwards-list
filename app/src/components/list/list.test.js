import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

import ListComponent from './list.component';
import ItemComponent from './item.component';
import SearchInput from '../inputs/search.component';

const testList = `<ul>
  <li>item 1</li>
  <li>item 2</li>
  <li>item 3</li>
  <li>item 4</li>
  <li>item 9</li>
  <li>item 10</li>
  <li>item 5</li>
  <li>item 7</li>
  <li>item 8</li>
  <li>item 6</li>
  </ul>
`;

test('List snapshot', () => {
  const component = renderer.create(
    <ListComponent list={testList} />
  );

  let tree = component.toJSON();
  
  expect(tree).toMatchSnapshot();
});
