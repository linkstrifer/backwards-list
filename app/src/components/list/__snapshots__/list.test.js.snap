// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List snapshot 1`] = `
<div>
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "space-between",
      }
    }
  >
    <input
      onInput={[Function]}
      placeholder="buscar"
    />
    <div>
      <label
        htmlFor="input-order"
        style={
          Object {
            "marginRight": "10px",
          }
        }
      >
        Ordenar:
      </label>
      <select
        id="input-order"
        onChange={[Function]}
      >
        <option
          defaultValue={true}
          value="asc"
        >
          a...z
        </option>
        <option
          value="des"
        >
          z...a
        </option>
      </select>
    </div>
  </div>
  <ul>
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
    <li />
  </ul>
</div>
`;
